cfgov/jinja2/v1/_includes/organisms/ask-search.html

```diff
@@ -43,11 +43,9 @@
 <div class="o-search-bar">
     <form method="get" action="{{ _('/ask-cfpb/search/') }}">
         {% if show_label %}
-        <label for="o-search-bar_query">
-            <{{ 'h3' if is_subsection else 'h2' }}>
-                {{ _('Search for your question') }}
-            </{{ 'h3' if is_subsection else 'h2' }}>
-        </label>
+        <h2 {{ 'class=h3' if is_subsection else '' }}>
+            {{ _('Search for your question') }}
+        </h2>
```

Comment:
I think this needs to stay switching the element itself instead of the style (so `{{ 'h3' if is_subsection else 'h2' }}`). On Ask answer pages, the outline right now looks like:

```
h1: {Main heading}
    h2: Don't see what you're looking for?
        h3: Browse related questions
        h3: Search for your question

I assume we want to keep that hierarchy on answer pages.

Reply:
Fixed in e3e9bd9